app/src/main/java/com/example/android/persistence/codelab/db/Loan.java

```diff
@@ -35,7 +36,9 @@
 @TypeConverters(DateConverter.class)
 public class Loan {
     // Fields can be public or private with getters and setters.
-    public @PrimaryKey String id;
+    public @PrimaryKey
```

Comment:
Keep the order like in the `Book` class: 
``` java
@PrimaryKey
@NonNull
public String id;
```

Reply:
I forgot to add files to the commit. Done